projects/plugins/jetpack/extensions/blocks/latex/save.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * React hook that is used to mark the block wrapper element.
+ * It provides all the necessary props like the class name.
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+
+/**
+ * The save function outputs the static markup for the rendered formula,
+ * with a data-latex attribute for possible front-end enhancement.
+ * @param {object} props            - Block props.
+ * @param {object} props.attributes - Block attributes.
+ * @return {object} The static markup for the rendered formula.
+ */
+export default function save( { attributes } ) {
+	const { latex = '' } = attributes;
+	return (
+		<div { ...useBlockProps.save( { className: 'jetpack-latex' } ) } data-latex={ latex }>
```

Comment:
Same as above, I think we could maybe do without this:


```suggestion
		<div { ...useBlockProps.save() } data-latex={ latex }>
```